docs/source/backends/hdf5.rst

```diff
@@ -97,10 +97,11 @@ Known Issues
 .. warning::
 
    Jul 23th, 2021 (`HDFFV-11260 <https://jira.hdfgroup.org/browse/HDFFV-11260>`__):
-   Collective HDF5 metadata reads became broken in 1.10.5.
-   Consider using 1.10.4 if you plan to enable the collective HDF5 metadata operations optimization in openPMD (``OPENPMD_HDF5_COLLECTIVE_METADATA=ON``).
-   Enabling this feature with a newer version will make HDF5 fall back to the individual metadata operations.
-   HDF5 plans to fix the issue in the upcoming 1.10.8+ and 1.12.2+ releases, but visit the issue tracker above to see the status of the bug fix.
+   Collective HDF5 metadata reads broke in 1.10.5.
+   Enabling collective HDF5 metadata operations (``OPENPMD_HDF5_COLLECTIVE_METADATA=ON``) with a newer version will make HDF5 fall back to the individual metadata operations.
```

Comment:
Changes seem ok, I am just concerned people will get confused as the next line points which versions have this fixed.

Reply:
Thank you for the review! Let me reorder the text if that makes more sense.

Reply:
updated, let me know what you think @jeanbez :)

Reply:
I believe it is clearer now! Thanks!